lib/tasks/dev.rake

```diff
@@ -1,3 +1,138 @@
-desc "Fill the database tables with some sample data"
-task({ :sample_data => :environment }) do
+namespace :dev do
+  desc "drops, creates, migrates, and adds sample data"
+  task reset: [
+         :environment,
+         "db:drop",
+         "db:create",
+         "db:migrate",
+         "dev:sample_data",
+       ]
+
+  desc "adds sample data for development environment"
+
+  task sample_data: [
+         :environment,
+         "dev:add_users",
+         "dev:add_grinders",
+         "dev:add_brewers",
+         "dev:add_coffees",
+       ] do
+    puts "done"
+  end
+
+  task add_users: :environment do
+    puts "add users"
+
+    names = ["jun", "thierry", "ian", "tony", "chase", "jesus", "andre", "derek", "nour", "alejandra", "ainsley", "gustavo"]
+
+    names.each do |name|
+      u = User.create(
+        email: "#{name}@example.com",
+        password: "password",
+        first_name: "#{name}",
+        last_name: Faker::Name.last_name,
+      )
+    end
+  end
+
+  task add_grinders: :environment do
+    puts "add grinders"
+
+    brands = ["Baratza", "Eureka", "1zpresso", "Turin"]
+    names = ["encore", "mignon", "k1 ultra", "df64", "df83"]
+    powers = ["electric", "power"]
+    burr_types = ["flat", "conical"]
+    burrs = ["stock", "diamond like coating", "high uniformity", "lab sweet"]
+
+    User.all.each do |user|
+      brand = brands.sample
+      name = names.sample
+      power = powers.sample
+      burr_type = burr_types.sample
+      burr = burrs.sample
+
+      g = Grinder.create(
```

Comment:
same thing here you should be able to delete g unless you call it for a specific function